bitswap/client/client.go

```diff
@@ -178,7 +187,9 @@ func New(parent context.Context, network bsnet.BitSwapNetwork, providerFinder ro
 		}
 	}
 	peerQueueFactory := func(ctx context.Context, p peer.ID) bspm.PeerQueue {
-		return bsmq.New(ctx, p, network, onDontHaveTimeout, bsmq.WithDontHaveTimeoutConfig(bs.dontHaveTimeoutConfig))
+		return bsmq.New(ctx, p, network, onDontHaveTimeout,
+			bsmq.WithDontHaveTimeoutConfig(bs.dontHaveTimeoutConfig),
+			bsmq.WithPerPeerSendDelay(bs.perPeerWantSendDelay))
```

Comment:
nit: bit odd that we end up  setting more broad "PerPeerSendDelay" to kinda narrowly-named "PeerWantSendDelay". Someone may think it only applies to Wants, but it will also aply to other things, no?

Could we rename `[With]PeerWantSendDelay` to  `[With]PerPeerSendDelay` so its named the same everywhere and there is no confusion?